### PR TITLE
Removes CTM email address handling

### PR DESCRIPTION
**Pull request by johnclary**
- https://github.com/cityofaustin/atd-data-tech/issues/12622

This PR removes a chunk of logic from this script and updates the README.

If you'd like to test this, you can follow the steps in the readme to run the script. You'll need to connect to VPN. The env vars can be found in "Knack HR App API Access" and "HR Banner API". 

☝️ Will get those 1pass entries cleaned up when we move this deployment to airflow v2.

One thing I'm not sure about is if the prefect flow will pull the latest image after we merge:
https://github.com/cityofaustin/atd-prefect/blob/main/flows/atd-knack-banner/knack_banner.py

I'd also like to deprecate the `master` branch and will do that once this is approved.

### CODE REVIEW

File: update_employees.py [hunk 1]
```diff
@@ -423,15 +372,20 @@ def main():
 
     result = {}
 
+    logging.info("Getting employee data from Banner...")
     records_hr_banner = get_employee_data()
-    employee_emails = get_emails_data()
-    records_hr_emails = update_emails(records_hr_banner, employee_emails)
-    records_mapped = map_records(records_hr_emails, FIELD_MAP, KNACK_APP_NAME)
+    # employee_emails = get_emails_data()
+    # records_hr_emails = update_emails(records_hr_banner, employee_emails)
```
Comment (chiaberry) [inline update_employees.py:378]: should this also be deleted?

File: README.md
Comment (chiaberry) [inline README.md:1]: thank you for all of this

**Comment by chiaberry** [comment]
I don't think the prefect flow will automatically pull the latest image, but I'm not 100% sure.

**Review by chiaberry — APPROVED** [review]
Thank you for taking care of this, and cleaning up the code and writing a README

File: update_employees.py (hunk 1 above)
Comment (johnclary) [reply update_employees.py]: ✅ yep thanks!

**Comment by johnclary** [comment]
I'm going to merge b/c the ETL is currently failing. 100% happy to incorporate all feedback once y'all have a chance to review.

**Comment by johnclary** [comment]
I [patched](https://github.com/cityofaustin/atd-knack-banner/commit/ff5fa24b94628b02f9aae5af0c26512c0a5b95a3) the image build, the action succeeded, i manually pulled the image on the prefect server, and manually triggered a successful flow run ✅ 😅 

I also deleted the `master` branch and set `production` as the default.

**Review by mddilley — COMMENTED** [review]
Just catching up on this issue since I wasn't super familiar with this process - thanks for fixing this up and adding to the readme. It stinks that it broke, but it is great that you found out that the .csv wasn't up to date either so you could simplify this script. 🚀

**Pull request merged**